### PR TITLE
Update turtlecoin-nodes.json

### PR DESCRIPTION
**Pull request by nodetrtlespain**

### CODE REVIEW

**Comment by brandonlehmann** [comment]
Node is not responding to public requests.

File: turtlecoin-nodes.json [hunk 1]
```diff
@@ -196,6 +196,13 @@
       "ssl": false,
       "cache": false
     },
+    {
+      "name": "Turtle Node Spain",
```
Comment (nodetrtlespain) [inline turtlecoin-nodes.json:207]: {
      
      
              "name": "Turtle Node EU",
      
      
              "url": "nodeturtlecoineu.ddns.net",
      
      
              "port": 5000,
      
      
              "ssl": false,
      
      
              "cache": false
      
      
            },

**Review by brandonlehmann — CHANGES_REQUESTED** [review]
One node is passing (EU) the other is not (Spain)

**Comment by brandonlehmann** [comment]
If you allowed maintainers to make edits to your PR then I could get one of these to pass through.

File: turtlecoin-nodes.json (hunk 1 above)
Comment (brandonlehmann) [reply turtlecoin-nodes.json:207]: The changes show BOTH nodes still in there. Pick one and remove the other.

**Pull request closed without merging**